config/dynamic-scaling-configuration.yaml

```diff
@@ -3,10 +3,31 @@
 #The following properties must be defined for each instance types compute node configuration:
 #   - max_compute_nodes: the maximum number of nodes needed to create an appropriate machine pool with the configuration needed by the autoscaler
 
+# For each supported cloud provider, the following properties must be provided for the machine type
+#   - cluster_wide_workload_machine_type: the instance type size to handle cluster wide workloads e.g ingress controllers, observability operators, etc 
+#   - kafka_workload_machine_type: the instance type size to handle the Kafka instance workload e.g brokers, kafka-admin API, etc
+
 ---
-developer:
-  compute_nodes_config:
-    max_compute_nodes: 3
-standard:
-  compute_nodes_config:
-    max_compute_nodes: 9
+# The version of OpenShift on newly auto created data plane clusters.
+# If the value is empty (default), then the latest version is going to be used. 
+new_data_plane_openshift_version: ""
```

Comment:
instead of `new_...` could we say `initial_*` ?

Reply:
Can we be more specific on what the "latest" version is?

Reply:
> The latest version will be up to OCM to choose it. KFM just leave the OCM field not configured in this case.

Can we document that?

Reply:
> The usage of new here is to say it is the data plane that's new not the version. So this reads like The openshift version applied to newly created data plane

I understand. The proposed change was to avoid that kind of confusion.

Reply:
> > The latest version will be up to OCM to choose it. KFM just leave the OCM field not configured in this case.
> 
> Can we document that?

```
# If the value is empty (default), then OpenShift Cluster Manager picks the latest version of OpenShift to be installed. 
``
How does that one sound?

Reply:
Let's improve it with doc :+1:

Reply:
done.